src/multi-selector.js

```diff
@@ -172,7 +173,7 @@ export default class MultiSelector extends React.Component {
         disabled: this.props.disableInput ? this.props.disableInput(e.target.value) : false,
       },
       () => {
-        this.inputChange(e.target.value);
```

Comment:
Since this is asynchronous, the `e` here is different than the `e` passed into the function. Here I'm fixing this bug.